README.md

```diff
@@ -50,8 +50,8 @@ Note that if you are using `bson` through the `mongodb` crate, you do not need t
 | `chrono-0_4` | Enable support for v0.4 of the [`chrono`](https://docs.rs/chrono/0.4) crate in the public API.              | n/a                | no      |
 | `uuid-0_8`   | Enable support for v0.8 of the [`uuid`](https://docs.rs/uuid/0.8) crate in the public API.                  | n/a                | no      |
 | `uuid-1`     | Enable support for v1.x of the [`uuid`](https://docs.rs/uuid/1.0) crate in the public API.                  | n/a                | no      |
-| `serde_with` | Enable [`serde_with`](https://docs.rs/serde_with/latest) integrations for `bson::DateTime` and `bson::Uuid` | serde_with         | no      |
-
+| `serde_with` | Enable [`serde_with`](https://docs.rs/serde_with/latest) 1.x integrations for `bson::DateTime` and `bson::Uuid`.| serde_with         | no      |
```

Comment:
I'm doing a terrible job reviewing for which I apologize!  Reading this made me go look and there are also [implementations](https://github.com/mongodb/bson-rust/blob/b243db19b74745fcf52eacd9e4d34077186186d5/src/datetime.rs#L460) for `crate::DateTime` which will need `serde_with_3` versions as well.